functions/v2/helloworld/hello_pubsub.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 Google LLC
```

Comment:
nit: new files use current year

```suggestion
// Copyright 2021 Google LLC
```